tests/suites/test_suite_chacha20.function

```diff
@@ -93,42 +93,37 @@ void chacha20_bad_params()
     size_t len = sizeof( src );
     mbedtls_chacha20_context ctx;
 
-    mbedtls_chacha20_init( NULL );
-    mbedtls_chacha20_free( NULL );
```

Comment:
Please keep a test for `free( NULL )` using `TEST_VALID_PARAM()`.

Reply:
(Same for the other modules.)

Reply:
How do you suggest to test this using `TEST_INVALID_PARAM` if `NULL` is not an invalid parameter to `mbedtls_xxx_free()`?

Reply:
@hanno-arm You may have already noticed, but @mpg was referring to `TEST_VALID_PARAM()` rather than `TEST_INVALID_PARAM()`. This commit https://github.com/ARMmbed/mbedtls/pull/1643/commits/54e7f312d33581a83e4dde3fbf5750ffbf5d322f makes the macro compiled in regardless of the parameter checking, so you may want to rebase this PR to incorporate that change.

Reply:
Thanks @k-stachowiak, and sorry @mpg for not reading your suggestion properly - I wasn't aware of the existence of `TEST_VALID_PARAM`.

Reply:
Addressed in ab133a7.

Reply:
@hanno-arm "I wasn't aware of the existence of TEST_VALID_PARAM." That's because you didn't re-review #1643 recently ;)